src/interfaces/file/input.vue

```diff
@@ -109,40 +93,50 @@ export default {
     };
   },
   computed: {
+    file() {
+      if (this.value !== null) return this.value;
+      if (this.values !== null && this.values.filesize && this.values.filename) return this.values;
+      return void 0;
```

Comment:
Prefer `undefined` over `void 0`

```suggestion
      return undefined;
```

Reply:
> The void operator is often used merely to obtain the undefined primitive value

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void

Reply:
Sure, your project, your style guide 😄
Just as a side-noe - minifiers/obfuscators still will convert `undefined` to `void 0`. Also `undefined` can be reassigned to another value in non-global function context, so in theory void <anything> is more secure. But as a matter of fact - it's only about the taste, never saw anybody reassigning `undefined` in real production code 😃

Reply:
> Just as a side-noe - minifiers/obfuscators still will convert undefined to void 0

Makes sense, saves a couple characters. My main goal is to keep Directus as approachable for external devs as possible; you see `void 0`  way less "in the wild" than `undefined`

> undefined can be reassigned to another value in non-global function context

This is also true, but also plain stupid. It's the same idea as that we could make an object method name a zero-width space ^^

![image](https://user-images.githubusercontent.com/9141017/58286895-bc986a00-7d7d-11e9-91ea-74d363c95694.png)

Reply:
> plain stupid

Couldn't agree more 😃 
And I get why `undefined` is better for third-party devs, but I blame community for that (imo) bad situation :<
That said I can resolve this conversation ;)